src/main.py

```diff
@@ -0,0 +1,13 @@
+from fastapi import FastAPI
+
+app = FastAPI()
+
+
+@app.get("/")
+def read_root():
+    return {"Hello": "World"}
+
+
+@app.get("/items/{item_id}")
+def read_item(item_id: int, q: str = None):
```

Comment:
```suggestion
def read_item(item_id: int, q: str | None = None):
```

https://qiita.com/Yossy_Hal/items/2feee8d6c91f9270f266